### DISCUSSION TITLE
How to enable hasura enterprise trial on azure container apps

### DISCUSSION THREAD

**Discussion by Code-Divine** [hasura/graphql-engine#10073] [Question]
I filled out the form and it told me to restart my container. In Azure container apps they are revisions, so I restarted the revision but nothing happens, I am still on on basic.

**Comment by manasag** [comment]
Hi. Self serve trial activation requires outbound connection to Hasura servers. Could you check if internet egress traffic is allowed on the Azure container app. Maybe this helps here https://learn.microsoft.com/en-us/azure/container-instances/container-instances-egress-ip-address#create-outbound-application-rule-on-the-firewall
If this doesn't work for you, please reach out to our sales for an offline license key instead.

**Comment by Code-Divine** [comment]
That is for container instances, I am using container apps. But anyway, I will contact support for offline key as you advised.
